conf/templates/handlebarswrapper-search-bar.txt

```diff
@@ -0,0 +1,31 @@
+import Handlebars from 'handlebars/dist/handlebars.runtime.min.js';
```

Comment:
is this file still used?

Reply:
To the best of my knowledge, we still need this, yes. It's the HBS run-time. I believe this is needed to actually render the pre-compiled templates properly at run-time.

Reply:
sorry I meant is handlebarswrapper-search-bar.txt still used? it looks like only handlebarswrapper.txt (not handlebarswrapper-search-bar.txt) is being used by gulp-wrap?

Reply:
Ahhh, gotcha gotcha. Nope, this can be deleted!